tests/test_wave_02.py

```diff
@@ -50,3 +50,15 @@ def test_get_tasks_sorted_desc(client, three_tasks):
             "is_complete": False,
             "title": "Answer forgotten email 📧"},
     ]
+
+
+def test_get_response_invalid_sort_query(client, three_tasks):
```

Comment:
Woo, test coverage!  :tada:

(This might seem like I'm being sarcastic but I'm genuinely a fan of people writing tests.)